javascript_nodejs/5.multiturn-prompts-bot/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "multiturn-prompts-bot",
+  "version": "1.0.0",
+  "description": "Bot Builder v4 multiturn prompts sample",
+  "main": "index.js",
+  "scripts": {
+    "start": "node ./index.js",
+    "watch": "nodemon ./index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "Microsoft",
+  "license": "MIT",
+  "dependencies": {
+    "botbuilder": "^4.0.0-preview1.2",
```

Comment:
This is where the problem started; please pull the latest bits from https://botbuilder.myget.org/gallery/botbuilder-v4-js-daily